newrelic_lambda_cli/cli/integrations.py

```diff
@@ -163,57 +171,55 @@ def uninstall(aws_profile, aws_region, aws_permissions_check, nr_account_id, for
 @add_options(AWS_OPTIONS)
 @click.option(
     "--enable-logs/--disable-logs",
-    default=False,
+    default=None,
     help="Determines if logs are forwarded to New Relic Logging",
-    show_default=True,
+    show_default=False,
 )
 @click.option(
     "--memory-size",
     "-m",
-    default=128,
+    default=None,
     help="Memory size (in MiB) for the log ingestion function",
     metavar="<size>",
-    show_default=True,
+    show_default=False,
     type=click.INT,
 )
-@add_options(NR_OPTIONS)
 @click.option(
     "--timeout",
     "-t",
-    default=30,
+    default=None,
```

Comment:
Why this change?

Reply:
If the CLI has a default value, we can't tell, later on, whether the user specified 30, or just didn't specify a value. For upgrade, an unspecified value now means, "Preserve the existing value". If they specified 120 at install time, then we want to make it easy to preserve that, rather than forcing the user to remember all their settings from the install when they upgrade.